scylla/tests/ccm_integration/ccm/mod.rs

```diff
@@ -1,8 +1,27 @@
 pub(crate) mod cluster;
 mod logged_cmd;
 pub(crate) mod node_config;
+
+use std::future::Future;
+use std::path::PathBuf;
+use std::sync::Arc;
 use std::sync::LazyLock;
+
+use cluster::Cluster;
+use cluster::ClusterOptions;
+use tokio::sync::Mutex;
 use tracing::info;
+
+pub(crate) static CLUSTER_VERSION: LazyLock<String> =
+    LazyLock::new(|| std::env::var("SCYLLA_TEST_CLUSTER").unwrap_or("release:6.2.2".to_string()));
+
+static TEST_KEEP_CLUSTER_ON_FAILURE: LazyLock<bool> = LazyLock::new(|| {
```

Comment:
:question: What is this going to be useful for?

Reply:
You can inspect the contents of the test tables, or the general state of the cluster, after the test fails.
With docker we had this automatically, now we don't.